scripts/ansible/roles/linux/intel/tasks/sgx-driver.yml

```diff
@@ -57,3 +57,9 @@
     state: started
     enabled: yes
   when: "'aesmd.service' in ansible_facts.services"
+
+- name: Set out-of-proc attestation by default
+  lineinfile:
+    path: /etc/environment
+    state: present
+    line: SGX_AESM_ADDR=1
```

Comment:
These lines make developers environment safe from the SGX in-kernel transition. However, they also make the developers **unaware** of the fact that their application users have to do the same.

I suggest we remove these settings.

@bodzhang any comment?

Reply:
It was actually Bo that suggested we put the environment variable into /etc/environment :)

Reply:
We want to encourage users to move to out-of-proc attestation. I think configure it as default is OK.